test/diamorphosis/diamorphosis.test.ts

```diff
@@ -59,4 +59,97 @@ describe('Diamorphosis Test', () => {
       config.honeybadger.environment.should.equal(process.env.NODE_ENV);
     });
   });
+
+  describe('should set json/console loggingvariables', () => {
+    let options;
+
+    beforeEach(() => {
+      options = {
+        diamorphosis: {
+          configFolder: './test/diamorphosis',
+          configPath: path.resolve('./test/diamorphosis/config'),
+          envFolder: path.resolve('./test/diamorphosis/env')
+        }
+      } as OrkaOptions;
+    });
+
+    afterEach(() => {
+      delete require.cache[require.resolve(options.diamorphosis.configPath)];
+    });
+
+    it('nothing set => console:true, json:false, styles:[] ', () => {
+      let config = {} as any;
+
+      diamorphosis(config, {} as OrkaOptions);
+
+      config.log.console.should.equal(true);
+      config.log.json.should.equal(false);
+      config.riviere.styles.length.should.equal(0);
+    });
+
+    it('console:not set, json:true, styles:[] => console:false, json:true, styles:["json"] ', () => {
+      let config = {
+        log: {
+          json: true
+        }
+      } as any;
+
+      diamorphosis(config, {} as OrkaOptions);
+
+      config.log.console.should.equal(false);
+      config.log.json.should.equal(true);
+      config.riviere.styles.length.should.equal(1);
+      config.riviere.styles[0].should.equal('json');
+    });
+
+    it('console:true, json:true, styles:[] => console:true, json:true, styles:["json"] ', () => {
```

Comment:
Can you add a test where instead of providing the config you provide env variables in process.env?
This way you can test the case I mention above

Reply:
Done